### PR TITLE
chore: fixes vanity import.

### LINKED ISSUE #844: ci: add check that vanity import comments exist
See https://github.com/elastic/apm-agent-go/pull/841#pullrequestreview-523192012. Like https://github.com/elastic/apm-agent-go/blob/master/scripts/check_goimports.sh, we could call https://github.com/jcchavezs/porto to see if changes are required and exit with an error if so.

### PR DESCRIPTION
**Pull request by jcchavezs**
This PR adds a check for vanity imports to be updated on CI.

Ping @axw

Closes #844

### CODE REVIEW

File: scripts/before_install.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@ if (go run scripts/mingoversion.go 1.12 &>/dev/null); then
   go install -v golang.org/x/lint/golint
   go install -v golang.org/x/tools/cmd/goimports
   go install -v github.com/elastic/go-licenser
+  go get -v github.com/jcchavezs/porto
```
Comment (jcchavezs) [inline scripts/before_install.sh:8]:
```suggestion
  go install -v github.com/jcchavezs/porto
```

File: Makefile [hunk 2]
```diff
@@ -85,3 +85,7 @@ model/marshal_fastjson.go: model/model.go
 
 scripts/Dockerfile-testing: $(wildcard module/*)
 	go generate ./scripts
+
+.PHONY: check-vanity-import
+check-vanity-import:
```
Comment (axw) [inline Makefile:90]: can you please move this up with the other conditional targets at line 15? and define an empty target for Go < 1.12

**Review by axw — CHANGES_REQUESTED** [review]
thanks! I'd just like to shuffle the Makefile change up to be with the other conditional "check-" targets please

**Comment by jcchavezs** [comment]
@axw I addressed your comments.

File: scripts/before_install.sh (hunk 1 above)
Comment (axw) [reply scripts/before_install.sh:8]: Any reason not to make that change? I would prefer to have consistency

**Review by axw — APPROVED** [review]
If we can change `go get` to `go install` for consistency that'd be great. Otherwise LGTM, thanks!

File: scripts/before_install.sh (hunk 1 above)
Comment (jcchavezs) [reply scripts/before_install.sh:8]: I think I did not commit it by mistake.

**Comment by axw** [comment]
Tests are failing but pretty certain that's unrelated, so merging. Thanks!

**Comment by axw** [comment]
> Tests are failing but pretty certain that's unrelated, so merging. Thanks!

Oops, looks like it's related to the testify upgrade actually. I'll sort it out.

**Pull request merged**